File: Java/gesbrasilburger/src/main/java/sn/brasilburger/entity/Burger.java
package sn.brasilburger.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@NoArgsConstructor
@Getter
@Setter
@AllArgsConstructor
public class Burger {
    private int id;
    private String nom;
    private Double prix;
    private String image;
    private Boolean etat = true;
    @Override
    public String toString(){
        return " Burger->[ID: " + id + ", Nom: " + nom + ", Prix: " + prix + "]";
    }
}

File: CSharp/brasilBurger/Models/Burger.cs
namespace brasilBurger.Models
{
    public class Burger
    {
        public int Id { get; set; }
        public string Nom { get; set; }
        public decimal Prix { get; set; }
        public string Image { get; set; }
        public bool Etat { get; set; }
        public ICollection<Menu> Menus { get; set; } = new List<Menu>();
    }
}